admin/common_auth/forms.py

```diff
@@ -15,3 +16,11 @@ def __init__(self, *args, **kwargs):
         super(CustomUserRegistrationForm, self).__init__(*args, **kwargs)
         self.fields['first_name'].required = True
         self.fields['last_name'].required = True
+
+    def clean_password1(self):
```

Comment:
While it is certainly good to enforce password length, doesn't Django already do this somewhere?